### PR TITLE
Update org.yaml after GC elections

### PR DESCRIPTION
**Pull request by DavidPHirsch**
Adding new GC Members to org

### CODE REVIEW

**Comment by mowies** [comment]
ok then let's leave it for now and see how it goes

File: config/keptn/org.yaml [hunk 1]
```diff
@@ -12,7 +12,8 @@ members_can_create_repositories: false
 admins:
   # governance committee
   - AnaMMedina21
-  - AloisReitbauer
```
Comment (mowies) [inline config/keptn/org.yaml:15]: alois needs to be added to the org members in order to be able to put him into the `emeriti` team

Comment (DavidPHirsch) [reply config/keptn/org.yaml:15]: done

File: config/keptn/org.yaml [hunk 2]
```diff
@@ -12,7 +12,8 @@ members_can_create_repositories: false
...
+  - salaboy
+  - thschue
```
Comment (mowies) [inline config/keptn/org.yaml:16]: thschue in turn needs to be remove from the members if he's in the admins 😅

**Review by thisthat — APPROVED** [review]

**Review by mowies — APPROVED** [review]

**Pull request merged**